### ISSUE TITLE
unable to chat with a user which is not initialized before.

### ISSUE THREAD

**Issue by abhirup-patra** [GetStream/stream-chat-android#522]
is there a way to chat with a user which is not initialized before?

**Comment by elevenetc** [comment]
What do you mean by initialised user? Is it an error message?

**Comment by abhirup-patra** [comment]
previously i used a different chat implementation, now using stream. the user is already present on the system but not initialized by login in any device. I want to chat with that user by creating a channel but unable to do so as the user is not registered with the stream chat. Previously in build 4.0.3 we just called the setuser method and it worked that way but after the inclusion of this chat domain i am not able to neither close the previous chat domain nor able to assign a new user to it. it crashes while doing so. I already created a thread with chat domain before but it is still not answered. Is there any other way around?
